### PR TITLE
Featured/new hive

### PR DESCRIPTION
**Pull request by dpopleton**
Minor updates to the core models. Improved relationships necessary for metadata-api to function.

### CODE REVIEW

File: src/python/ensembl/core/models.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ class AttribType(Base):
     name = Column(String(255), nullable=False, server_default=text("''"))
     description = Column(Text)
 
-    seq_region_attrib = relationship("SeqRegionAttrib", back_populates="attrib_type")
+    seq_region_attributes = relationship("SeqRegionAttrib", back_populates="attrib_type")
```
Comment (JAlvarezJarreta) [inline src/python/ensembl/core/models.py:106]: This attribute name change is going to affect [at least another repository](https://github.com/Ensembl/ensembl-genomio/blob/48087e0c9f17cfedd93321d764130400b292baba/src/python/ensembl/io/genomio/seq_region_dumper.py#L31), so unless this change is required, I would suggest leaving it as it was. Furthermore, other Python repositories seem to give it the same previous name, e.g. [ensembl-api](https://github.com/Ensembl/ensembl-api/blob/710082a1d891f89e30322164572878a94c84ec67/src/python/ensembl/core/data/model/models.py#L978).

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]
It is always nice to see some code cleanup :relaxed:

File: src/python/ensembl/core/models.py (hunk 1 above)
Comment (dpopleton) [reply src/python/ensembl/core/models.py]: Not even sure why I changed that. Changed back

File: src/python/ensembl/core/models.py [hunk 2]
```diff
@@ -1970,29 +1765,8 @@ class SeqRegionAttrib(Base):
...
-#          "attrib_type_id",
-#          ForeignKey("attrib_type.attrib_type_id"),
-#          nullable=False,
-#          server_default=text("'0'"),
-#      ),
-#      Column("value", Text, nullable=False, index=True),
-#      Index("type_val_idx", "attrib_type_id", "value"),
-#      Index("region_attribx", "seq_region_id", "attrib_type_id", "value", unique=True),
-#  )
+    attrib_type = relationship("AttribType", back_populates="seq_region_attributes")
```
Comment (JAlvarezJarreta) [inline src/python/ensembl/core/models.py:1768]: This needs to be reverted as well:
```suggestion
    attrib_type = relationship("AttribType", back_populates="seq_region_attrib")
```

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]

**Review by JAlvarezJarreta — APPROVED** [review]

**Pull request merged**